modules/tinymce/src/plugins/table/test/ts/browser/ui/TableDialogTest.ts

```diff
@@ -366,4 +366,52 @@ describe('browser.tinymce.plugins.table.TableDialogTest', () => {
     await TableTestUtils.pClickDialogButton(editor, false);
     editor.options.unset('table_appearance_options');
   });
+
+  it('TINY-8758: width should be retained when changing border.', async () => {
```

Comment:
```suggestion
  it('TINY-8758: width should be retained when changing border', async () => {
```
There shouldn't be a fullstop here.